fdbserver/include/fdbserver/DataDistributionTeam.h

```diff
@@ -127,9 +128,13 @@ class TeamSelect {
 	}
 
 	bool operator==(const TeamSelect& tmpTeamSelect) { return value == tmpTeamSelect.value; }
+	void setForRelocateShard(ForRelocateShard tmpForRelocate) { forRelocateShard = tmpForRelocate; }
+
+	bool isForRelocateShard() const { return forRelocateShard == ForRelocateShard::True; }
 
 private:
 	Value value;
+	ForRelocateShard forRelocateShard;
```

Comment:
Since this is not being made a part of the enum, is there a benefit to having this in the `TeamSelect` class rather than just a part of `GetTeamRequest`?

Reply:
It has to change many places if we are going to add `ForRelocateShard ` into `GetTeamRequest`, and it might not worth the complexity to do that since we would only set `ForRelocateShard=True` in one place.